index.html

```diff
@@ -170,6 +171,30 @@ <h4 class="connectmywork"> LET’S CONNECT</h4>
         </ul>
   </div>
   </section>
-  
+  <section class="Workssection">
+    <form action="https://formspree.io/f/mwkzayzq" method="post" class="cardcontact">
+        
+        <div class="background1">
+            <h2  class="contactme" >Contact me </h2>
+        <p class="sectioncontact"> If you have an application you are interested 
+            in developing, a feature that you need built or a project that 
+            needs coding. I’d love to help with it  </p>
+        </div>
+            <div class="form-group">
+                 <input class="texform" type="text"   id="Name"    classe="form-control" placeholder="Enter Your Name" minlength="4" maxlength="30" required />
+            </div>
+            <div class="form-group">          
+                <input class="texform"  type="email"  id="email"  name="email"   placeholder="Enter Your Email"  required  />
+            </div>
+            <div class="form-group2">
+                <input class="texform2"  name="comment"  id="comment"   cols="50"   rows="6"  placeholder="Write your message here..."  required  />
```

Comment:
Great job @DamenaM 🤝

- [ ] Kindly change this from an `<input>` field into a `<textarea>`.
- [ ] Kindly add a `maxlength="500"` property to this `<textarea>` as specified in the project requirements.